### ISSUE TITLE
Header and footer text color too dark in dark mode

### ISSUE THREAD

**Issue by digitalhobbit** [one-aalam/astro-ink#29]
When I switch to dark mode, the header and footer text remains black, making it barely readable. E.g. the line that reads "Crisp, minimal, personal blog theme for Astro".

It looks like there's some code in "global.css" to handle this, but as far as I can tell, this doesn't seem to be working. I also saw a possibly related open pull request.

(Beautiful theme otherwise - thanks!)

**Comment by drewbitt** [comment]
Yes, the contrast is very low and not accessible. 1.43:1 contrast for the black text on the dark background. You can edit `header__desc` in `Header.astro` and `post-preview__desc` in `PostPreview.astro` to change the font colors on the index page.

**Comment by one-aalam** [comment]
Hey, @digitalhobbit and @drewbitt This issue is fixed now.

Thank you for your patience.
